### PR TITLE
[pose-detection]Add model quality table for BlazePose.

### PR DESCRIPTION
**Pull request by lina128**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/741)

### CODE REVIEW

File: pose-detection/src/blazepose_mediapipe/README.md [hunk 1]
```diff
@@ -140,3 +141,16 @@ There is a difference of how things are loaded between the two runtimes. For the
 | Lite model | 10.41MB | 1.91s |
 | Full model | 13.8MB | 1.91s |
 | Heavy model | 34.7MB | 4.82s |
+
+## Model Accuracy
+This table compares MediaPipe BlazePose accuracy against other state-of-the-art
+pose solutions. For more detail, see the [article](https://google.github.io/mediapipe/solutions/pose#pose-estimation-quality).
+| Method | Yoga | Yoga | Dance | Dance | HIIT | HIIT |
+| --- | --- | --- | --- | --- | --- | --- |
```
Comment (bazarevsky) [inline pose-detection/src/blazepose_mediapipe/README.md:149]: Please add links to PCK@0.2 and mAP definitions. Original table refers them. For mAP https://cocodataset.org/#keypoints-eval . For PCK@0.2 https://github.com/cbsudux/Human-Pose-Estimation-101#percentage-of-correct-key-points---pck

**Review by lina128 — COMMENTED** [review]
Thank you for the review Valentin!

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/741#-:-MaBE9_61mXCdKvVd07z:b3wh0zw)* status: 0 of 1 approvals obtained (waiting on @bazarevsky, @ivan-grishchenko, and @pyu10055)

___
*[pose-detection/src/blazepose_mediapipe/README.md, line 149 at r1](https://reviewable.io/reviews/tensorflow/tfjs-models/741#-MaB8EeMGX7Jz3_lD4lW-r1-149:-MaBE9-_B93nO-yeONiG:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-models/blob/4cfffdc2710cb8eb4c50b3d221ac40414f1558ab/pose-detection/src/blazepose_mediapipe/README.md#L149)):*
<details><summary><i>Previously, bazarevsky (v.bazarevsky) wrote…</i></summary><blockquote>

Please add links to PCK@0.2 and mAP definitions. Original table refers them. For mAP https://cocodataset.org/#keypoints-eval . For PCK@0.2 https://github.com/cbsudux/Human-Pose-Estimation-101#percentage-of-correct-key-points---pck
</blockquote></details>

Done.

File: pose-detection/src/blazepose_mediapipe/README.md [hunk 2]
```diff
@@ -140,3 +141,16 @@ There is a difference of how things are loaded between the two runtimes. For the
 | Lite model | 10.41MB | 1.91s |
 | Full model | 13.8MB | 1.91s |
 | Heavy model | 34.7MB | 4.82s |
+
+## Model Accuracy
+This table compares MediaPipe BlazePose accuracy against other state-of-the-art
```
Comment (bazarevsky) [inline pose-detection/src/blazepose_mediapipe/README.md:146]: Please add the description about verticals and rename Accuracy to Quality (Accuracy is another well known metric, so let's not confuse readers). 

Pose Estimation Quality
To evaluate the quality of our models against other well-performing publicly available solutions, we use three different validation datasets, representing different verticals: Yoga, Dance and HIIT. Each image contains only a single person located 2-4 meters from the camera. To be consistent with other solutions, we perform evaluation only for 17 keypoints from COCO topology.

Comment (bazarevsky) [reply pose-detection/src/blazepose_mediapipe/README.md:146]: Link to topology:  https://cocodataset.org/#keypoints-2020

**Review by bazarevsky — APPROVED** [review]

**Pull request merged**